elastic/kibana
# [ResponseOps][Alerts] Don't show empty state in grouping component while first loading

## Summary

Makes the loading state and empty state mutually exclusive in the grouping component to avoid showing the empty state when first loading the groups data.

## To verify

1. Create one or more O11y rules that fire alerts
2. Open the O11y > Alerts page
3. Toggle on grouping
4. Reload the page (possibly after activating network throttling)
5. Verify that while the loading indicator is shown, the empty state is not and viceversa

## References

Fixes #190954
